docs/setup/settings.asciidoc

```diff
@@ -325,11 +342,16 @@ default is `true`.
 `server.socketTimeout:`:: *Default: "120000"* The number of milliseconds to wait before closing an
 inactive socket.
 
-`server.ssl.certificate:` and `server.ssl.key:`:: Paths to the PEM-format SSL
-certificate and SSL key files, respectively.
+`server.ssl.certificate:` and `server.ssl.key:`:: Paths to a PEM-encoded X.509 certificate and its private key, respectively. These are used
+when enabling SSL for inbound requests from web browsers to the Kibana server.
 
-`server.ssl.certificateAuthorities:`:: List of paths to PEM encoded certificate
-files that should be trusted.
+NOTE: These settings cannot be used in conjunction with `server.ssl.keystore.path`.
+
+`server.ssl.certificateAuthorities:`:: Paths to one or more PEM-encoded X.509 certificates. This may consist of a root certificate authority
+(CA), and one or more intermediate CAs, which make up a trusted certificate chain for Kibana. This is used when a web browser creates an SSL
+connection with the Kibana server; the certificate chain is sent to the browser along with the end-entity certificate to establish trust.
+This is also used to determine if client certificates should be trusted when PKI authentication is enabled. In addition to this setting,
+trusted certificates may be specified via `server.ssl.keystore.path` and/or `server.ssl.truststore.path`.
```

Comment:
The prior documentation for `server.ssl.certificateAuthorities` was not entirely accurate. This setting serves a dual purpose -- I don't really like it, but that's how it operates today. If we wanted to change this behavior and split it into two settings, we'd probably have to muck around Hapi and customize the verification code for client SSL certificates.